src/use_case/chat/ChatMessageDataAccessInterface.java

```diff
@@ -0,0 +1,10 @@
+package use_case.chat;
+
+import entity.Message;
+import java.util.List;
+
+public interface ChatMessageDataAccessInterface {
+  void save(Message message);
```

Comment:
Should this return the status of the save request?

Reply:
Yep, created a new issue to add a UI for failed network requests #12, and we can do this as part of that issue